pagopa/io-app
# fix: [IOBP-36,IOBP-37] Remove read status from transactions

## Short description
This PR removes the read status from the transaction list in wallet section and the unread transactions counter badge from bottom navigation bar.

| Before | After |
| -- | -- |
| <video src="https://github.com/pagopa/io-app/assets/6160324/e49ddbfb-547e-4ad5-b786-aafc806adb74.mov" />   |  <video src="https://github.com/pagopa/io-app/assets/6160324/20418464-30f4-4660-aa06-1d1c0021286c.mov" /> |




## List of changes proposed in this pull request
- Removed `readTransactions` reducer and selector
- Removed saga to get unread transaction state after transaction load
- Removed blue dot (`isNew` prop) from `DetailedlistItemComponent` in `TransactionList`
- Removed unread messages counter from `WalletTabIcon.tsx`

## How to test
In the wallet section of the app, load new transaction with the dedicated button in the bottom section of the screen. Check that:
- No blue dot is displayed in list items
- No counter badge is displayed in the bottom navigation bar
